getting-started.ipynb

```diff
@@ -0,0 +1,591 @@
+        "id": "dVSVAT0OEcF1"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Now let us define the training and validation datasets (as\n",
+        "[torch.utils.data.DataLoader](https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader))\n",
+        "and store them in `train_loader` and `val_loader` respectively. We have\n",
+        "used the [MNIST](https://pytorch.org/vision/stable/datasets.html#mnist)\n",
+        "dataset for ease of understanding.\n"
+      ],
+      "metadata": {
+        "id": "DDIW2zedEcF3"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "source": [
+        "data_transform = Compose([ToTensor(), Normalize((0.1307,), (0.3081,))])\n",
+        "\n",
+        "train_loader = DataLoader(\n",
+        "    MNIST(download=True, root=\".\", transform=data_transform, train=True), batch_size=128, shuffle=True\n",
+        ")\n",
+        "\n",
+        "val_loader = DataLoader(\n",
+        "    MNIST(download=True, root=\".\", transform=data_transform, train=False), batch_size=256, shuffle=False\n",
+        ")"
+      ],
+      "outputs": [],
+      "metadata": {
+        "id": "PFNgx_-TEcF4"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Finally, we will specify the optimizer and the loss function:"
+      ],
+      "metadata": {
+        "id": "VC9BUtWXEcF6"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 7,
+      "source": [
+        "optimizer = torch.optim.RMSprop(model.parameters(), lr=0.005)\n",
+        "criterion = nn.CrossEntropyLoss()"
+      ],
+      "outputs": [],
+      "metadata": {
+        "id": "6VkGmtVZEcF7"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "And we’re done with setting up the important parts of the project.\n",
+        "PyTorch-Ignite will handle all other boilerplate code as we will see\n",
+        "below. Next we have to define a trainer engine by passing our model,\n",
+        "optimizer and loss function to\n",
+        "[`create_supervised_trainer`](https://pytorch.org/ignite/generated/ignite.engine.create_supervised_trainer.html),\n",
+        "and an evaluator engine by passing Ignite’s out-of-the-box\n",
+        "[metrics](https://pytorch.org/ignite/metrics.html#complete-list-of-metrics)\n",
+        "and the model to\n",
+        "[`create_supervised_evaluator`](https://pytorch.org/ignite/v0.4.5/generated/ignite.engine.create_supervised_evaluator.html#create-supervised-evaluator)\n",
+        ":"
+      ],
+      "metadata": {
+        "id": "cb-ak9gEEcF7"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 10,
+      "source": [
+        "trainer = create_supervised_trainer(model, optimizer, criterion, device)\n",
+        "\n",
+        "val_metrics = {\n",
+        "    \"accuracy\": Accuracy(),\n",
+        "    \"nll\": Loss(criterion)\n",
```

Comment:
Why do we use `nll` here? resnet output layer is linear, that's why we use crossentropy (logsoftmax+nll), can we call it nll in this case?
@vfdev-5

Reply:
Right, thanks @trsvchn let's call it "loss"